app/grapefruit/app-dev.toml

```diff
@@ -6,3 +6,43 @@ inherit = "base.toml"
 features = ["uart8"]
 uses = ["uart8"]
 interrupts = {"uart8.irq" = "usart-irq"}
+
+# Ereport stuff
+[tasks.packrat]
+stacksize = 1040
+features = ["ereport"]
+
+[tasks.snitch]
+name = "task-snitch"
+priority = 6
```

Comment:
It looks like `net` is priority 3 in the Grapefruit image, does it matter that `snitch` isn't right above it?

Reply:
ah, i think i just copied this from some other app.toml, it should probably be bumped up. good catch.

`packrat` had priority 3 on gimletlet, as well, and i bumped it up, i should do that on grapefruit too --- i forgor